tests/test_georaster.py

```diff
@@ -1814,6 +1854,116 @@ def test_polygonize(self) -> None:
+            raster.proximity(distance_unit=distance_unit, target_values=target_values)
+            .data.data.squeeze()
+            .astype("float32")
+        )
+
+        # The results should be the same in all cases
+        try:
+            # In some cases, the proximity differs slightly (generally <1%) for complex settings
+            # (Landsat Raster with target of 112)
+            # It looks like GDAL might not have the right value,
+            # so this particular case is treated differently in tests
+            if target_values is not None and target_values[0] == 112 and raster.filename is not None:
+
+                # Get index and number of not almost equal point (tolerance of 10-4)
+                ind_not_almost_equal = np.abs(gdal_proximity - geoutils_proximity) > 1e-04
+                nb_not_almost_equal = np.count_nonzero(ind_not_almost_equal)
+                # Check that this is a minority of points (less than 0.5%)
+                assert nb_not_almost_equal < 0.005 * raster.width * raster.height
+
+                # Replace these exceptions by zero in both
+                gdal_proximity[ind_not_almost_equal] = 0.0
+                geoutils_proximity[ind_not_almost_equal] = 0.0
+                # Check that all the rest is almost equal
+                assert np.allclose(gdal_proximity, geoutils_proximity, atol=1e-04, equal_nan=True)
+
+            # Otherwise, results are exactly equal
+            else:
+                assert np.array_equal(gdal_proximity, geoutils_proximity, equal_nan=True)
+
+        # For debugging
+        except Exception as exception:
+
+            import matplotlib.pyplot as plt
+
+            # Plotting the xdem and GDAL attributes for comparison (plotting "diff" can also help debug)
+            plt.subplot(121)
+            plt.imshow(gdal_proximity)
+            # plt.imshow(np.abs(gdal_proximity - geoutils_proximity)>0.1)
+            plt.colorbar()
+            plt.subplot(122)
+            plt.imshow(geoutils_proximity)
+            # plt.imshow(raster.data.data == 112)
+            plt.colorbar()
+            plt.show()
+
+            # ind_not_equal = np.abs(gdal_proximity - geoutils_proximity)>0.1
+            # print(gdal_proximity[ind_not_equal])
+            # print(geoutils_proximity[ind_not_equal])
+
+            raise exception
+
+    def test_proximity_parameters(self) -> None:
+        """
+        Test that new (different to GDAL's) proximity parameters run.
+        No need to test the results specifically, as those rely entirely on the previous test with GDAL,
+        and tests in rasterize and shapely.
+        #TODO: Maybe add one test with an artificial vector to check it works as intended
+        """
+
+        # -- Test 1: with self's Raster alone --
+        raster1 = gu.Raster(self.landsat_b4_path)
+        prox1 = raster1.proximity()
+
+        # The raster should have the same extent, resolution and CRS
+        assert raster1.crs == prox1.crs
```

Comment:
Hmm, we should have a function that's like `r1.equal_grid(r2)` which is true if the CRS, transform and shape is the same. Would be pretty useful! I might add it as an issue later.

Reply:
Or `is_aligned()`, `aligned()`, `same_place()` or something. Suggestions are welcome!

Reply:
Speak of the devil: #202 !

Reply:
Done with the name `equal_georeferenced_grid`, added tests and replaced by this where relevant!